parameters.go

```diff
@@ -183,10 +189,24 @@ func convertNamedValuesToSparkParams(values []driver.NamedValue) []*cli_service.
 		} else {
 			sparkParamType = sqlParam.Type.String()
 		}
-		sparkParam := cli_service.TSparkParameter{Name: &sqlParam.Name, Type: &sparkParamType, Value: sparkValue}
+
+		var sparkParamName *string
+		if sqlParam.Name != "" {
```

Comment:
what if `sqlParam.Name` is `nil`

Reply:
The type of `Parameter.Name` is `string` and not `*string`, so it cannot be `nil`. And therefore we use an empty string as an indicator of ordinal parameter (it's not possible to have named parameter with empty name anyway)